app/services/submission/plant_trial_finalizer.rb

```diff
@@ -47,10 +117,19 @@ def create_scoring
           trait_descriptor = get_nth_trait_descriptor(trait_mapping[col_index])
           rollback(1) unless trait_descriptor
 
+          trait_score_attributes = common_data
```

Comment:
Not sure if it is a problem here but `trait_score_attributes = common_data.dup` would be safer. Otherwise you may unwillingly assign `technical_replicate_number`.